### DISCUSSION TITLE
CiLocks

### DISCUSSION THREAD

**Discussion by hari5556** [tegal1337/CiLocks#41] [General]
Hello guys I have a doubt 
In parrot os after i installed cilocks
That 
Tool executed and when I give option it shows error like (adb command not found) what can i do i connected my phone with pc using usb
1.Is it necessary to download Android adb in parrot os or not necessary 
2.I used my mobile hotspot for laptop in this case is it necessary to connect phone and pc with same other wifi 
Pls someone give me a solution

**Comment by ZAYNO430** [comment]
Je veux débloquer mon iPhone 12pro max bloqué
